.github/workflows/create_release.yml

```
@@ -0,0 +1,86 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Caller Workflow
+on: 
```

```suggestion
on:
```

nit: trailing spaces.

could you format all new yml files with an auto formatter? Thanks!

---

Done